docs/api/enums.rst

```diff
@@ -242,6 +242,58 @@ of :class:`enum.Enum`.
         The system message denoting that an context menu command was executed.
 
         .. versionadded:: 2.0
+    .. attribute:: auto_moderation_action
+
+        The system message denoting an action by automod.
+
+        .. versionadded:: 2.4
```

Comment:
NVM, this should be 2.3 as it was present prior to this PR.
```suggestion
        .. versionadded:: 2.3
```

Reply:
damn